### DISCUSSION TITLE
Swagger files with POST instead of GET

### DISCUSSION THREAD

**Discussion by filipjakov** [kubb-labs/kubb#1146] [Q&A] [answered]
I have an API implementation that uses POST instead of GET for some requests. I want to turn these POST requests to `useQuery`, not `useMutation`. 

I tried something overrides in kubb.config.ts:

```tsx
override: [
	// {
	// 	type: 'operationId',
	// 	pattern: '.*_get.*',
	// 	options: {
	// 		mutate: false,
	// 	},
	// },
	{
		type: 'operationId',
		pattern: '.*_get.*',
		options: {
			templates: {
				// @ts-expect-error
				mutation: Query.templates,
			},
		},
	},
],
```

but failed as expected.
Has anyone faced a similar issue and has ideas on how to approach this?

**Comment by stijnvanhulle** [comment, marked as answer]
For that you can use methods, see https://www.kubb.dev/plugins/swagger-tanstack-query/#methods
```
override: [
	// {
	// 	type: 'operationId',
	// 	pattern: '.*_get.*',
	// 	options: {
	// 		mutate: false,
	// 	},
	// },
	{
		type: 'operationId',
		pattern: '.*_get.*',
		options: {
			mutate: {
                            variablesType: 'hook',
                             methods: ['get',  'post', 'put', 'delete' ],
                         },
		},
	},
],
```

**Comment by filipjakov** [comment]
@stijnvanhulle tried lts (3.6.5) and it looks like this feature is not available any more? intended, regression or did i misconfigure something?
